### PR TITLE
fix: populate missing 'enabled' service field from file

### PR DESCRIPTION
**Pull request by GGabriele**
Without this decK is failing at parsing the 'enabled' field
for services entities.

### CODE REVIEW

**Review by rainest — APPROVED** [review]
For reference, https://github.com/Kong/kong/commit/1339494f9cfc8f08433db91a7ce29443ddbe2483

**Review by rainest — CHANGES_REQUESTED** [review]
Code :+1:, can you add a pending release changelog entry for this?

**Comment by hbagdi** [comment]
Please add a test.

**Comment by GGabriele** [comment]
> Code 👍, can you add a pending release changelog entry for this?

I still need to get used to that :( added now

> Please add a test.

done

File: file/reader_test.go [hunk 1]
```diff
@@ -149,3 +149,38 @@ func TestReadKongStateFromStdin(t *testing.T) {
...
+	assert.Equal(kong.Upstream{
+		Name:      kong.String("upstream1"),
+		Algorithm: kong.String("round-robin"),
+	},
+		c.Upstreams[0].Upstream)
+
+	assert.Equal(kong.Target{
+		Target: kong.String("198.51.100.11:80"),
+	},
+		c.Upstreams[0].Targets[0].Target)
```
[the remark this answers is not in the document]
Comment (GGabriele) [reply file/reader_test.go]: sounds good

https://github.com/Kong/deck/pull/677/commits/882c51342f41bad6570325a9683de47d3105b8db

**Review by rainest — APPROVED** [review]

**Pull request merged**